File: C-Sharp-part-2/03.Methods/05. Larger than neighbours.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

class largerThanNeighbours
{
    static int LargerNumber(int[] numbers)
    {
        int count = 0;
        for (int i = 1; i < numbers.Length - 1; i++)
        {
            if (numbers[i] > numbers[i - 1] && numbers[i] > numbers[i + 1])
            {
                count++;
            }
        }
        return count;
    }

    static void Main()
    {
        int arr = int.Parse(Console.ReadLine());
        int[] numbers = new int[arr];
        string[] input = Console.ReadLine().Split(' ');
        for (int i = 0; i < input.Length; i++)
        {
            numbers[i] = int.Parse(input[i]);
        }

        Console.WriteLine(LargerNumber(numbers));
    }
}

File: JavaScript-Fundamentals/05.Functions/05. Larger than neighbours.js
function larger(input) {
    "use strict";

    var arrLength = input[0];
    var arr = input[1].split(' ').map(Number);
    var count = 0;

    for (var i = 1; i < arr.length - 1; i += 1) {
        //for (var i = 0; i < input[0]; i += 1)  
        if (arr[i] > arr[i - 1] && arr[i] > arr[i + 1]) {
            count += 1;
        }
    }
    console.log(count);
}

// larger(['6', '-26 -25 -28 31 2 27']);
